### ISSUE TITLE
A few DRY and KISS issues. Focus and clear button issues.

### ISSUE THREAD

**Issue by romans-adi** [romans-adi/todo-app#4]
This project follows the best HTML, CSS, and JavaScript practices.
Overall, the project seems to follow the DRY (Don't Repeat Yourself), KISS (Keep It Simple, Stupid), and YAGNI (You Ain't Gonna Need It) principles quite well. However, there may be some parts of the code that could be further improved.

**For DRY:**

- Instead of checking conditionals twice https://github.com/romans-adi/todo-app/blob/53c500fd21fe5e25b59214703ba5c3deba0bbd68/src/modules/render.js#L36, you could check it only once and remove the inner conditional.

- The same practice could be used for [this](https://github.com/romans-adi/todo-app/blob/53c500fd21fe5e25b59214703ba5c3deba0bbd68/src/modules/render.js#L37-L39C18) code. Instead of calling rendering and saving functions https://github.com/romans-adi/todo-app/blob/53c500fd21fe5e25b59214703ba5c3deba0bbd68/src/modules/render.js#L59, code could be simplified by removing the repeating actions and creating a separate function (for example, updateTask) instead.

**For KISS and YAGNI:**

In terms of KISS, the code appears to be relatively simple and straightforward in its implementation.  However, there are areas where the code could be simplified by removing unnecessary elements.

- The  ripple https://github.com/romans-adi/todo-app/blob/53c500fd21fe5e25b59214703ba5c3deba0bbd68/src/modules/render.js#L128 and https://github.com/romans-adi/todo-app/blob/53c500fd21fe5e25b59214703ba5c3deba0bbd68/src/modules/render.js#L20  parts may not be essential for the core functionality of the app, but they could be useful for improving the user experience.

- The `removeBtn` event listener https://github.com/romans-adi/todo-app/blob/53c500fd21fe5e25b59214703ba5c3deba0bbd68/src/modules/render.js#L90 is added every time the description element is focused on, which could lead to unnecessary overhead. Instead, the listener could be added once when the `removeBtn` element is created and then simply toggled between the hidden and visible states when the description element is focused on and blurred.

- _The `checkboxToggle` function is a separate file that is imported and used in the render function. While this function is necessary to toggle the completion state of a task, it could be simplified and integrated into the render function itself to make the code more straightforward.
**Since it was a project requirement, you can ignore this suggestion.**_

**Other issues:**

:exclamation:  Focusing after adding a new task is not working :exclamation:

Please, check that your function https://github.com/romans-adi/todo-app/blob/53c500fd21fe5e25b59214703ba5c3deba0bbd68/src/modules/render.js#L40 is working properly and not blocked by calling the render function.

:exclamation: Clear completed tasks button is not working properly :exclamation:

Please, check your styles for particular elements https://github.com/romans-adi/todo-app/blob/53c500fd21fe5e25b59214703ba5c3deba0bbd68/src/styles.css#L250.

**Comment by romans-adi** [comment]
All the issues have been solved! Thank you for your suggestions :smiley_cat:
